spec/factories/consent_step.rb

```diff
@@ -1,6 +1,6 @@
 FactoryBot.define do
   factory :consent_step do
-    sequence(:order) { |n| n }
+    sequence(:order) { |n| n + 1 }
```

Comment:
Some tests were failing because the `order==1st` step requires a non-empty `tour_video` field. This numbers the order fields starting at 2.